cfme/tests/services/test_catalog.py

```diff
@@ -5,31 +5,55 @@
 import cfme.tests.configure.test_access_control as tac
 from cfme import test_requirements
 from cfme.services.catalogs.catalog import CatalogsView
+from cfme.services.catalogs.catalog import DetailsCatalogView
+from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.update import update
 
 pytestmark = [test_requirements.service, pytest.mark.tier(2)]
 
 
 @pytest.mark.rhel_testing
 @pytest.mark.sauce
-def test_catalog_crud(appliance):
+def test_catalog_crud(request, appliance):
     """
     Polarion:
         assignee: nansari
         casecomponent: Services
         initialEstimate: 1/8h
         tags: service
     """
+    # Create Catalog
     catalog_name = fauxfactory.gen_alphanumeric()
     cat = appliance.collections.catalogs.create(name=catalog_name, description='my catalog')
+    request.addfinalizer(cat.delete_if_exists)
 
     view = cat.create_view(CatalogsView)
     assert view.is_displayed
-    view.flash.assert_success_message('Catalog "{}" was saved'.format(catalog_name))
+    if appliance.version >= '5.11':
+        saved_message = f"Catalog was saved"
+    else:
+        saved_message = f'Catalog "{catalog_name}" was saved'
+    view.flash.assert_success_message(saved_message)
 
+    # Edit Catalog
     with update(cat):
         cat.description = 'my edited description'
+
+    view.flash.assert_success_message(saved_message)
+
+    view = navigate_to(cat, 'Edit')
+    view.cancel_button.click()
+    view = cat.create_view(DetailsCatalogView)
+    assert view.wait_displayed()
+    view.flash.assert_message(f'Edit of Catalog "{catalog_name}" was cancelled by the user')
+    assert cat.description == "my edited description"
+
+    # Delete Catalog
     cat.delete()
+    view = cat.create_view(CatalogsView)
+    view.flash.assert_success_message(
+        f'Catalog "{cat.description}": Delete successful')
```

Comment:
@niyazRedhat why does it use the description in 5.10 and 5.11, instead of the catalog name.

Reply:
BZ 1765107 was written